### ISSUE TITLE
LoadUFOfamily: autosave issue with Use Versions ticked

### ISSUE THREAD

**Issue by n7s** [silnrsi/glyphs-scripts#1]
If Use Versions is ticked In Glyphs -> Preferences -> User Settings on Glyphs 2.4.2 (987) there is an autosave error message: 
![autosave-issue](https://cloud.githubusercontent.com/assets/4881499/23956597/5c66a1ca-099d-11e7-8347-ca8e73b6775b.png)

**Comment by jvgaultney** [comment]
Right - this is because Glyphs can open, edit, and save UFOs without converting them to standard Glyphs files. However since masters/instances are not supported in Glyphs the autosave (to UFO) won't work. The fix is to have the import script temporarily save the file in Glyphs format. Then autosave would not complain anymore. However you have to be vary careful to not treat that temporary Glyphs-format file as the only source file. You would still have to export the UFOs as before (for which I hope to write a script).

**Comment by n7s** [comment]
To make things simpler and continue being UFO-centric then we could also consider turning off the autosave feature, no? Is that something we have been using extensively in the past?

**Comment by jvgaultney** [comment]
Yes, I'd suggest just turning it off, but I will look into the implications of temporarily saving in Glyphs format as well.
